### PR TITLE
Updated READMEs for the examples - Batch 3

### PR DESCRIPTION
**Pull request by andreasnaoum**
Updated READMEs for the examples: 
- Signed Distance Fields
- ROS Nodes
- Segment Anything Model
- RGBD
- Structure for motion
- RRT star
- Eigen OpenCV (C++)
- VRS (C++)
- Minimal Example
- Clock
- Multithreading
- Multiprocessing
- Plots

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [rerun.io/viewer](https://rerun.io/viewer/pr/5784)
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/5784?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/5784?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5784)
- [Docs preview](https://rerun.io/preview/27ab45ba41c1efa935b04e6f9d4fcc3babe03e1d/docs) 
- [Examples preview](https://rerun.io/preview/27ab45ba41c1efa935b04e6f9d4fcc3babe03e1d/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: examples/cpp/eigen_opencv/README.md [hunk 1]
```diff
@@ -17,3 +15,58 @@ You can find the example at <https://github.com/rerun-io/cpp-example-opencv-eige
...
+
+```cpp
+rec.log("world/points_from_matrix", rerun::Points3D(points3d_matrix));
+```
+
+## Pinhole Camera
+A pinhole camera is logged to "world/camera" using the [`Pinhole`](https://www.rerun.io/docs/reference/types/archetypes/pinhole) archetype.
+Additionally, the 3D transformation of the camera, including its position and orientation, is logged using the [`Transform3D`](https://www.rerun.io/docs/reference/types/archetypes/transform3d) archetype.
+```cpp
+rec.log("world/camera", rerun::Pinhole::from_focal_length_and_resolution({500.0, 500.0}, {640.0, 480.0}));
```
Comment (emilk) [inline examples/cpp/eigen_opencv/README.md:48]: This line is way too long

```suggestion
rec.log(
    "world/camera",
    rerun::Pinhole::from_focal_length_and_resolution({500.0, 500.0}, {640.0, 480.0})
);
```

File: examples/cpp/eigen_opencv/README.md [hunk 2]
```diff
@@ -17,3 +15,58 @@ You can find the example at <https://github.com/rerun-io/cpp-example-opencv-eige
   <source media="(max-width: 1024px)" srcset="https://static.rerun.io/cpp-example-opencv-eigen/2fc6355fd87fbb4d07cda384ee8805edb68b5e01/1024w.png">
   <source media="(max-width: 1200px)" srcset="https://static.rerun.io/cpp-example-opencv-eigen/2fc6355fd87fbb4d07cda384ee8805edb68b5e01/1200w.png">
 </picture>
+
+This is a minimal CMake project that shows how to use Rerun in your code in conjunction with [Eigen](https://eigen.tuxfamily.org/) and [OpenCV](https://opencv.org/).
+
+
+# Used Rerun Types
```
Comment (emilk) [inline examples/cpp/eigen_opencv/README.md:22]: We've recently decided to not use Title Casing for markdown headers. So we only Capitalize names, like `Rerun`
```suggestion
# Used Rerun types
```

File: examples/cpp/vrs/README.md [hunk 3]
```diff
@@ -5,17 +5,78 @@ tags = ["2D", "3D", "vrs", "viewer", "C++"]
...
 -->
+[//]: # (> VRS is a file format optimized to record & playback streams of sensor data, such as images, audio samples, and any other discrete sensors &#40;IMU, temperature, etc&#41;, stored in per-device streams of time-stamped records.)
 
-
-This is an example that shows how to use [Rerun](https://github.com/rerun-io/rerun)'s C++ API to log and view [VRS](https://github.com/facebookresearch/vrs) files.
-
-> VRS is a file format optimized to record & playback streams of sensor data, such as images, audio samples, and any other discrete sensors (IMU, temperature, etc), stored in per-device streams of time-stamped records.
-
-You can find the example at <https://github.com/rerun-io/cpp-example-vrs>.
+[//]: # (You can find the example at <https://github.com/rerun-io/cpp-example-vrs>.)
```
Comment (emilk) [inline examples/cpp/vrs/README.md:10]: What syntax is this?

`[//]: #` is news to me. I can't find anything about it on https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

And what's with the `&#40;`?

File: examples/cpp/vrs/README.md [hunk 4]
```diff
@@ -5,17 +5,78 @@ tags = ["2D", "3D", "vrs", "viewer", "C++"]
...
+# Logging and Visualizing with Rerun
+
+The visualizations in this example were created with the following Rerun code:
+
+## 3D Arrows
+```cpp
+void IMUPlayer::log_accelerometer(const std::array<float, 3>& accelMSec2) {
+        _rec->log(_entity_path + "/accelerometer", rerun::Arrows3D::from_vectors({accelMSec2}));
+        // ... existing code for scalars ...
+}
```
Comment (emilk) [inline examples/cpp/vrs/README.md:39]: four spaces for indentation

File: examples/cpp/vrs/README.md [hunk 5]
```diff
@@ -5,17 +5,78 @@ tags = ["2D", "3D", "vrs", "viewer", "C++"]
...
+```
+
+## Scalars 
+```cpp
+void IMUPlayer::log_accelerometer(const std::array<float, 3>& accelMSec2) {
+        // ... existing code for Arrows3D ...
```
Comment (emilk) [inline examples/cpp/vrs/README.md:45]:
```suggestion
        // … existing code for Arrows3D …
```

File: examples/python/multiprocessing/README.md [hunk 6]
```diff
@@ -13,8 +14,83 @@ thumbnail_dimensions = [480, 480]
...
 Demonstrates how rerun can work with the python `multiprocessing` library.
 
+# Used Rerun Types
+[`Boxes2D`](https://www.rerun.io/docs/reference/types/archetypes/boxes2d), [`TextLog`](https://www.rerun.io/docs/reference/types/archetypes/text_log)
+
+# Logging and Visualizing with Rerun
+This example demonstrates how to use the rerun with `multiprocessing` to log data from multiple processes to the same Rerun viewer.
+It starts with the definition of the function for logging, the `task`, followed by typical usage of Python's `multiprocessing` library.
+
+The function `task` is decorated with @rr.shutdown_at_exit. This decorator ensures that data is flushed when the task completes, even if the normal atexit-handlers are not called at the termination of a multiprocessing process.
```
Comment (emilk) [inline examples/python/multiprocessing/README.md:27]:
```suggestion
The function `task` is decorated with `@rr.shutdown_at_exit`. This decorator ensures that data is flushed when the task completes, even if the normal `atexit`-handlers are not called at the termination of a multiprocessing process.
```

File: examples/python/plots/README.md [hunk 7]
```diff
@@ -16,8 +16,123 @@ channel = "main"
...
+        # ... existing code ...
+
+        rr.log(
+            "curves/parabola",
+            rr.Scalar(f_of_t),
+            rr.SeriesLine(width=width, color=color),
+        )
+ ```
+## Trig
+
```
Comment (emilk) [inline examples/python/plots/README.md:72]:
```suggestion

## Trig
```

File: examples/cpp/vrs/README.md (hunk 3 above)
Comment (emilk) [reply examples/cpp/vrs/README.md:10]: They don't show up in https://github.com/rerun-io/rerun/blob/andreasnaoum-examples-batch3/examples/cpp/vrs/README.md so are they comments, or what?

**Review by emilk — CHANGES_REQUESTED** [review]
Looks good except for the weird `[//]: #` that I don't understand

File: examples/cpp/vrs/README.md (hunk 3 above)
Comment (andreasnaoum) [reply examples/cpp/vrs/README.md:10]: Yes, they are comments. I will finalize it now with spell check, lint and also clear the comments

File: examples/cpp/eigen_opencv/README.md (hunk 2 above)
Comment (andreasnaoum) [reply examples/cpp/eigen_opencv/README.md:22]: All the examples should change then. Should I make it in that batch and later with a different pull request change all the other ones?

Comment (emilk) [reply examples/cpp/eigen_opencv/README.md]: I can batch change it after merge

File: examples/cpp/vrs/README.md (hunk 3 above)
Comment (emilk) [reply examples/cpp/vrs/README.md]: `<!-- … -->` is the more common markdown comment style, from HTML

File: examples/cpp/vrs/README.md [hunk 8]
```diff
@@ -6,16 +6,79 @@ thumbnail = "https://static.rerun.io/vrs-viewer/28da92ebc2f0bccd5cf904314d2f8b0b
...
+
+# Logging and visualizing with Rerun
+
+The visualizations in this example were created with the following Rerun code:
+
+## 3D arrows
+```cpp
+void IMUPlayer::log_accelerometer(const std::array<float, 3>& accelMSec2) {
+        _rec->log(_entity_path + "/accelerometer", rerun::Arrows3D::from_vectors({accelMSec2}));
+        // … existing code for scalars …
```
Comment (emilk) [inline examples/cpp/vrs/README.md:35]: We should use four spaces for indentation everywhere

File: examples/python/multiprocessing/README.md [hunk 9]
```diff
@@ -13,8 +14,81 @@ thumbnail_dimensions = [480, 480]
...
+def task(child_index: int) -> None:
+    rr.init("rerun_example_multiprocessing")
+
+    rr.connect()
+
+    title = f"task_{child_index}"
+    rr.log(
+        "log",
+        rr.TextLog(
+            f"Logging from pid={os.getpid()}, thread={threading.get_ident()} using the rerun recording id {rr.get_recording_id()}"
```
Comment (emilk) [inline examples/python/multiprocessing/README.md:40]:
```suggestion
            f"Logging from pid={os.getpid()}, thread={threading.get_ident()} using the Rerun recording id {rr.get_recording_id()}"
```

File: examples/python/rgbd/README.md [hunk 10]
```diff
@@ -16,9 +16,77 @@ build_args = ["--frames=300"]
...
+```python
+rr.log("world/camera/image/rgb", rr.Image(img_rgb).compress(jpeg_quality=95))
+```
+
+## Depth image
+
+Pinhole camera is utilized for achieving a 3D view and camera perspective through the use of the [`Pinhole`](https://www.rerun.io/docs/reference/types/archetypes/pinhole).
+
+```python
+
```
Comment (emilk) [inline examples/python/rgbd/README.md:52]:
```suggestion
```

File: examples/python/ros_node/README.md [hunk 11]
```diff
@@ -56,6 +83,39 @@ ros2 launch nav2_bringup tb3_simulation_launch.py headless:=False
...
+[//]: # (As described in the nav demo, use the rviz window to initialize the pose estimate and set a navigation goal.)
+
+[//]: # ()
+[//]: # (You can now connect to the running ROS system by running:)
+
+[//]: # (```)
+
+[//]: # (python3 examples/python/ros_node/main.py)
+
+[//]: # (```)
```
Comment (emilk) [inline examples/python/ros_node/README.md:121]: Use `<!-- … -->` for comments - then they can also span multiple lines

File: examples/python/signed_distance_fields/README.md [hunk 12]
```diff
@@ -1,23 +1,100 @@
...
+This example illustrates the visualization of the results obtained from generating Signed Distance Fields (SDFs) for arbitrary meshes using both traditional methods and the approach described in the [DeepSDF paper](https://arxiv.org/abs/1901.05103).
+DeepSDF introduces a learned continuous representation of shapes using SDFs, enabling high-quality shape representation, interpolation, and completion from partial and noisy 3D input data.
+This novel approach offers improved performance and reduced model size compared to previous methods.
+The generated SDFs help with accurate 3D reconstruction and visualization.
+
+# Logging and visualizing with Rerun
+
+The visualizations in this example were created with the following Rerun code:
+
+## 3D Asset
```
Comment (emilk) [inline examples/python/signed_distance_fields/README.md:34]:
```suggestion
## 3D asset
```

**Review by emilk — APPROVED** [review]
LGTM, but please use four spaces for indenting the code everywhere. There is currently a mix of 4 and 8

File: examples/cpp/vrs/README.md (hunk 3 above)
Comment (andreasnaoum) [reply examples/cpp/vrs/README.md]: I used the keyboard shortcut Cmd + "/" for commenting, and that happened. It was also weird for me tbh

**Comment by emilk** [comment]
I can take this over.

Please next time break up your work into more, smaller PRs

**Pull request merged**